cfme/configure/access_control.py

```diff
@@ -379,7 +330,79 @@ def description(self):
         return self.credential.principal
 
 
-@navigator.register(User, 'All')
+@attr.s
+class UserCollection(BaseCollection):
+
+    ENTITY = User
+
+    def simple_user(self, userid, password):
+        creds = Credential(principal=userid, secret=password)
+        return self.instantiate(name=userid, credential=creds)
+
+    def create(self, name=None, credential=None, email=None, group=None, cost_center=None,
+               value_assign=None, cancel=False):
+        """ User creation method
+
+        Args:
+            name: Name of the user
+            credential: User's credentials
+            email: User's email
+            group: User's group for assigment
+            cost_center: User's cost center
+            value_assign: user's value to assign
+            appliance: appliance under test
+            cancel: True - if you want to cancel user creation,
+                    by defaul user will be created
+
+        Throws:
+            RBACOperationBlocked: If operation is blocked due to current user
+                not having appropriate permissions OR update is not allowed
+                for currently selected role
+        """
+        if self.appliance.version < "5.8":
+            user_blocked_msg = "Userid has already been taken"
+        else:
+            user_blocked_msg = ("Userid is not unique within region {}".format(
+                self.appliance.server.zone.region.number))
+
+        user = self.instantiate(
+            name=name, credential=credential, email=email, group=group, cost_center=cost_center,
+            value_assign=value_assign
+        )
+
+        view = navigate_to(self, 'Add')
+        view.fill({
+            'name_txt': user.name,
+            'userid_txt': user.credential.principal,
+            'password_txt': user.credential.secret,
+            'password_verify_txt': user.credential.verify_secret,
+            'email_txt': user.email,
+            'user_group_select': getattr(user.group, 'description', None)
+        })
+
+        if cancel:
+            view.cancel_button.click()
+            flash_message = 'Add of new User was cancelled by the user'
+        else:
+            view.add_button.click()
+            flash_message = 'User "{}" was saved'.format(self.name)
```

Comment:
**Required**

self is `UserCollection` here, this needs to be `user.name` instead of `self.name`

Reply:
bugger